communication/slack_channels.md

```diff
@@ -0,0 +1,78 @@
+# Slack Channels
+
+Please find a list here of Slack channels at source{d}.
+
+Slack channels are encouraged to be open for everyone to join. Private channels for professional reasons are highly discouraged (have fun with private personal slack channels). We encourage people to have discussions in open/public channels rather than direct messages when the information is relevant to multiple people for a limited amount of time. If a piece of information should be preserved, it should always go to the guide or company repo. 
+
+### General channels
+
+* `#general`
+  * Company wide announcements, company-related or generally important topics
+* `#offtopic`
+  * Almost anything goes, the place to share interesting or funny things and keep other channels clean and on topic
+* `#officechat`
+  * Physical office related discussions (SF & Madrid)
+* `#offtopic-`
+  * Targeted offtopic channels. Search in Slack for #offtopic- and you'll find different offtopic channels like #offtopic-music
+* `#osd`
+  * `#osd` stands for [Open Source Days](../talent/open_source_days.md), people share what they'll be working on during their OSD and share the results/demos
+* `#twitter`
+  * Source{d} mentions and tweets
+
+### Team channels
+
+* `#eng-team`
+* `#product-team`
+* `#dev-infrastructure`
+* `#dev-retrieval`
+* `#dev-language-analysis`
+* `#dev-machine-learning`
+* `#dev-applications`
+
+### Daily standup channels
+
+This is where the daily standups happen.
+
+* `#su-general` 
+  * For those of us whose team is too small to warrant its own channel, e.g. CEO
+* `#su-product`
+* `#su-infrastructure`
+* `#su-data-retrieval`
+* `#su-language-analysis`
+* `#su-machine-learning`
+* `#su-applications`
+
+### Public channels that are shared with the community slack
+
+Anyone can join [here](https://join.slack.com/t/sourced-community/shared_invite/enQtMjc4Njk5MzEyNzM2LTFjNzY4NjEwZGEwMzRiNTM4MzRlMzQ4MmIzZjkwZmZlM2NjODUxZmJjNDI1OTcxNDAyMmZlNmFjODZlNTg0YWM)
```

Comment:
Is this link permanently valid?
If so, could we use http://bit.ly/src-d-community instead?